### PR TITLE
feat: replace blockchain users to identities

### PR DESCRIPTION
**Pull request by shumkov**

### CODE REVIEW

File: lib/test/createIdentity.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  const { items: [input] } = await dapiClient.getUTXO(address.toString());
+
+  const lockTransaction = new Transaction();
+
+  lockTransaction.from(input)
+    .addBurnOutput(10000, publicKeyHash)
+    .change(faucetAddress)
+    .fee(668)
+    .sign(privateKey);
+
```
Comment (QuantumExplorer) [inline lib/test/createIdentity.js]: The same key pair is being used for the input address as for the identity. I'm guessing this was not done on purpose.

Comment (shumkov) [reply lib/test/createIdentity.js]: Good catch! Thank you!

Comment (shumkov) [reply lib/test/createIdentity.js]: @antouhou I forgot this part. So we agreed to use different private keys for a lock transaction and for identity what we going to create, right?

Comment (antouhou) [reply lib/test/createIdentity.js]: Yep, as it's one of the best practices in cryptography - never reuse the same key for two different purposes

File: lib/test/createIdentity.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+/**
+ *
+ * @param {DAPIClient} dapiClient
+ * @param {Address} faucetAddress
+ * @param {PrivateKey} faucetPrivateKey
+ * @param {Address} address
+ * @return {Promise<string>}
+ */
+async function fundAddress(dapiClient, faucetAddress, faucetPrivateKey, address) {
+  const { items: inputs } = await dapiClient.getUTXO(faucetAddress.toString());
```
Comment (thephez) [inline lib/test/createIdentity.js]: How is this going to work once dashevo/dapi/pull/218 removes `getUTXO`?

Comment (shumkov) [reply lib/test/createIdentity.js]: We brought it back.

**Pull request merged**